src/HtmlAgilityPack.Shared/HtmlNode.Encapsulator.cs

```diff
@@ -809,7 +817,14 @@ public sealed class XPathAttribute : Attribute
         /// <summary>
         /// The methode of output
         /// </summary>
-        public ReturnType NodeReturnType { get; set; }
+        public ReturnType NodeReturnType
+        {
+            get => _nodeReturnType; set
```

Comment:
Linebreak before the `set` keyword, please ;-)

Reply:
Thanks.